hosts

```diff
@@ -221,3 +221,9 @@ jenkins ansible_ssh_host=115.146.87.250
 ansible_host_key_checking=False
 ansible_user=ubuntu
 ansible_ssh_extra_args='-o StrictHostKeyChecking=no'
+
+[gms_webservers]
+site.usegalaxy.org.au  ansible_connection=ssh  ansible_host=115.146.85.213  ansible_user=ubuntu
```

Comment:
galaxy-site ansible_connection=ssh ansible_host=115.146.85.213 ansible_user=ubuntu internal_ip=192.168.205.199

Reply:
(edited - I initially wrote galaxy-site but galaxy-media is the name of the VM)

Reply:
Where'd you get the internal IP from?

Reply:
Justin got it from Michael. I'm not sure how to look for it with a command. @jlqfab do you know a way?

Reply:
I got it from Michael. There are ways of scanning the internal network (which will probably trigger security alerts). Easiest way to ensure the internal IP is correct it attempt an ssh from galaxy to the hosts (don't actually need to login).

Reply:
I did an `ip addr show` and that seemed to provide one amongst all the stdout

Reply:
```
ubuntu@galaxy-labs:~$ ip a
:
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 8942 qdisc fq_codel state UP group default qlen 1000
    link/ether fa:16:3e:27:94:f0 brd ff:ff:ff:ff:ff:ff
    altname enp0s3
    altname ens3
    inet 192.168.205.41/24 metric 100 brd 192.168.205.255 scope global dynamic eth0
       valid_lft 24331sec preferred_lft 24331sec
    inet6 fe80::f816:3eff:fe27:94f0/64 scope link 
       valid_lft forever preferred_lft forever
```

Reply:
```
ubuntu@galaxy-media:~$ ip a
   :
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 8942 qdisc fq_codel state UP group default qlen 1000
    link/ether fa:16:3e:48:d4:64 brd ff:ff:ff:ff:ff:ff
    altname enp0s3
    altname ens3
    inet 192.168.205.199/24 metric 100 brd 192.168.205.255 scope global dynamic eth0
       valid_lft 24146sec preferred_lft 24146sec
    inet6 fe80::f816:3eff:fe48:d464/64 scope link 
       valid_lft forever preferred_lft forever
```

Reply:
@neoformit see above for confirmation of internal IP addresses

Reply:
Thanks, yes looks right